docs/remote-development/vscode-ssh.md

```diff
@@ -0,0 +1,381 @@
+    ![](https://seneca-cdot-telescope.s3.amazonaws.com/vscode-ssh/2021-10-26+13_56_54-Get+Started+-+Visual+Studio+Code.png)
+    ![](https://seneca-cdot-telescope.s3.amazonaws.com/vscode-ssh/2021-10-26+13_58_26-.png)
+
+## Setting up your AWS credentials
+
+1. Open up a terminal in Visual Studio Code (hotkey on Windows: `Ctrl + backtick`). You should see that you're logged in as something like `ubuntu@ip-172.31.23.4`.
+
+2. Install `unzip`. We will need this to install the AWS CLI
+
+```
+$ sudo apt install unzip
+```
+
+3. Install the latest version of the AWS CLI
+
+```
+$ curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip"
+unzip awscliv2.zip
+sudo ./aws/install
+```
+
+4. Verify the AWS CLI installation
+
+```
+$ aws --version
+
+aws-cli/2.3.0 Python/3.8.8 Linux/5.4.0-1045-aws exe/x86_64.ubuntu.20 prompt/off
+```
+
+5. Remove the `awscliv2.zip` file and `aws` directory
+
+```
+rm awscliv2.zip
+rm -rf aws
+```
+
+6. Configure your AWS credentials
+
+```
+aws configure
+```
+
+7. Currently, everything is set as None so enter your credentials for your AWS IAM user.
+
+```
+AWS Access Key ID [None]: ****************764G
+AWS Secret Access Key [None]: ****************qBbe
+Default region name [None]: us-east-2
+Default output format [None]:
+```
+
+## Installing Docker and Docker-Compose
+
+### Install Docker Engine - Community Edition
+
+1. Update the apt package index:
+
+```
+sudo apt-get update
+```
+
+2. Install packages to allow apt to use a repository over HTTPS:
+
+```
+sudo apt-get install \
+    ca-certificates \
+    curl \
+    gnupg \
+    lsb-release
+```
+
+3. Add Docker’s official GPG key:
+
+```
+curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo gpg --dearmor -o /usr/share/keyrings/docker-archive-keyring.gpg
+```
+
+5. Use the following command to set up the stable repository:
+
+```
+echo \
+  "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/docker-archive-keyring.gpg] https://download.docker.com/linux/ubuntu \
+  $(lsb_release -cs) stable" | sudo tee /etc/apt/sources.list.d/docker.list > /dev/null
+```
+
+6. Update the apt package index again:
+
+```
+sudo apt-get update
+```
+
+7. Install the latest version of Docker Engine community:
+
+```
+sudo apt-get install docker-ce docker-ce-cli containerd.io
+```
+
+8. Add your user to the Docker group ([source](https://docs.docker.com/install/linux/linux-postinstall/)):
+
+```
+sudo usermod -aG docker $USER
+```
+
+9. Activate the changes to groupss
+
+```
+newgrp docker
+```
+
+10. Now run docker as a service on your machine, on startup:
+
+- Enable docker on startup: `sudo systemctl enable docker`
+- Disable docker on startup: `sudo systemctl disable docker`
+
+### Install Docker-Compose
+
+1. Run to download the current stable version of Docker-Compose:
+
+```
+sudo curl -L "https://github.com/docker/compose/releases/download/1.29.2/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
```

Comment:
Same question here... https://docs.docker.com/engine/reference/commandline/compose/

Reply:
I just installed Docker on a brand new Ubuntu machine to SSH to on AWS and the `docker compose` command is not available. 

```
ubuntu@ip-172-31-19-163:~/telescope$ docker

Usage:  docker [OPTIONS] COMMAND
...
Management Commands:
  app*        Docker App (Docker Inc., v0.9.1-beta3)
  builder     Manage builds
  buildx*     Build with BuildKit (Docker Inc., v0.6.3-docker)
  config      Manage Docker configs
  container   Manage containers
  context     Manage contexts
  image       Manage images
...
```